scripts/update-message-registry.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import argparse
+from pathlib import Path
+from typing import NamedTuple, Sequence
```

Comment:
I recommend using [dataclasses](https://docs.python.org/3/library/dataclasses.html), they have nicer semantics than named tuples.

Reply:
Ok, I left `Version` out, but I figured how to implement order & use it as an iterable. Will do on next update.

Reply:
Fixed